libs/dh/esett/feature-metering-gridarea-imbalance/src/lib/dh-metering-gridarea-imbalance.component.ts

```diff
@@ -183,4 +195,48 @@ export class DhMeteringGridAreaImbalanceComponent implements OnInit {
   handlePageEvent({ pageIndex, pageSize }: PageEvent): void {
     this.pageMetaData$.next({ pageIndex, pageSize });
   }
+
+  download() {
+    this.isDownloading = true;
+    this.queryVariables$
+      .pipe(
+        switchMap(
+          ({ filters, documentIdSearch, sortMetadata }) =>
+            this._apollo.watchQuery({
```

Comment:
suggestion: for simple request -> response action, `this._apollo.query(...)` can be used instead.

Reply:
fixed